flash/core/data/transforms.py

```diff
@@ -37,8 +42,10 @@ def __init__(self, keys: Union[str, Sequence[str]], *args):
         self.keys = keys
 
     def forward(self, x: Mapping[str, Any]) -> Mapping[str, Any]:
+        if not isinstance(x[DataKeys.INPUT], Tensor):
+            x[DataKeys.INPUT] = to_tensor(x[DataKeys.INPUT])
```

Comment:
Definitely shouldn't do this. INPUT may not always be an image (could be text for example). We should just manually include a `ToTensor` transform in the input transforms that need it

Reply:
note that this is a deprecated approach and kept only for legacy reasons...